nixos/modules/system/boot/stage-1.nix

```diff
@@ -128,6 +128,13 @@ let
         copy_bin_and_libs ${pkgs.e2fsprogs}/sbin/resize2fs
       ''}
```

Comment:
maybe something like:

```
${optionalString (any (fs: fs.autoResize && (lib.hasPrefix "ext" fs.fsType)) fileSystems)
```

but it could be also the case that only ext4 supports resizing, in which case the check can be simplified

Reply:
I think ext3 supports resizing too, and for ext2 to not support it would be illogical, as it is a common feature. Many filesystems support it.

I'll include your conditional in the next commit.